### PR TITLE
hotfix - getting rid of sudos

### PR DESCRIPTION
**Pull request by RobertLucian**
By doing this, we can install the script_tools repo/package with the right privileges as specified by the install command.
```
# with root privileges
sudo sh -c "curl -kL dexterindustries.com/update_tools | bash"
```
```
# with user privileges
curl -kL dexterindustries.com/update_tools | bash
```

### CODE REVIEW

File: functions_library.sh [hunk 1]
```diff
@@ -41,17 +41,17 @@ feedback() {
 change_branch() {
   # first and only parameter is the branch to checkout
 
-  # -z tests for zero length.
-    if [ -z ${1+x} ]; then
-        echo "Working from main branch.";
-    else
+  # -z tests for zero length.
```
Comment (johnisanerd) [inline functions_library.sh]: For the love of all holy, please, turn off your white space editing.

Comment (johnisanerd) [reply functions_library.sh]: It's not a big deal, but it makes reviewing edits a lot easier.

**Comment by johnisanerd** [comment]
@RobertLucian have you tested this on DexterOS yet?

File: functions_library.sh (hunk 1 above)
Comment (RobertLucian) [reply functions_library.sh]: There are no mods on the `functions_library.sh` file, so I can't see what you're seeing.

Second, I don't have any white space editing option thing going on my editor. I don't do space-
editing, I always prefer tabs.

Comment (johnisanerd) [reply functions_library.sh]: Seems to have disappeared.

**Comment by RobertLucian** [comment]
I have tested it on my version of DexterOS and partly on Raspbian For Robots. I'll be doing more tests anyway, because I have to bring changes to all repos.

File: functions_library.sh (hunk 1 above)
Comment (RobertLucian) [reply functions_library.sh]: I did yesterday night a reverse-commit so maybe you were seeing the old mods.

**Comment by CleoQc** [comment]
Just so you know, historically speaking, the Pi required sudo for a lot of stuff, accessing the pins for example. That's why there are sudos all over the place.

**Pull request merged**